.gitignore

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
Did you add the .env file to the .gitignore? It seems that it is not here. Please let me know if I didn't look for it well

Reply:
Now it is :)